samza-kv/src/main/scala/org/apache/samza/storage/kv/BaseKeyValueStorageEngineFactory.scala

```diff
@@ -143,7 +143,7 @@ trait BaseKeyValueStorageEngineFactory[K, V] extends StorageEngineFactory[K, V]
 
     // create the storage engine and return
     // TODO: Decide if we should use raw bytes when restoring
-    val keyValueStorageEngineMetrics = new KeyValueStorageEngineMetrics(storeName, registry)
+    val keyValueStorageEngineMetrics = new KeyValueStorageEngineMetrics(storeName, registry, jobContext.getConfig)
```

Comment:
Isn't the code below sufficient to disable timers? IIRC it returns the same value for each timer update which reduces the overhead.

Reply:
Thanks, it seems this is sufficient, will close this PR.